### ISSUE TITLE
[Bug] The old redact icon is still used instead of the new one 

### ISSUE THREAD

**Issue by dredav** [PDFTron/webviewer-ui#974]
**WebViewer version**
8.4.1

**The current behavior**
- The new redaction icon and the old one are used both in different places

**The expected behavior**
- Only the new icon should be used for consistency's sake

**Steps to reproduce**
- Go to https://www.pdftron.com/webviewer/demo/
- Select the toolgroup "Redact"
- Take a look to the new redacht icon
- Mark some text in the document
- Take a look to the old redaction icon which is now visible in the popup

See also the highlighted areas here:
![image](https://user-images.githubusercontent.com/731073/164683273-393d6e05-457f-4cc7-afe7-96f004d17f95.png)

**Comment by bollain** [comment]
Hi @dredav,

Thanks for catching this! I have made a PR to fix this. Once it is merged I will update this issue.

**Comment by bollain** [comment]
Hi @dredav,

The fix has now been merged into the 8.4 branch. You can get the latest build from our nightly stable builds page, or directly from NPM.
https://www.pdftron.com/documentation/web/faq/webviewer-nightly-build/#npm

Thanks again and let us know if you have any other issues or questions.
